src/interface/meta.thrift

```diff
@@ -88,8 +88,8 @@ struct ExecResp {
 // Graph space related operations.
 struct CreateSpaceReq {
     1: string space_name,
-    2: i32 parts_num,
-    3: i32 replica_factor,
+    2: optional i32 parts_num,
```

Comment:
Default field is enough in this case.

Reply:
I'm not sure whether it's OK if a non-optional field is not set, is that allowed?

Anyway, the present implementation of meta client set this field blindly. There is no such thing _default_ or _optional_.

Maybe this is a special case where there are values could be used as a default to represent NULL or not set. But we must be careful about cases there aren't such values.

The thrift proto on this struct in upstream has been modified, so this change should be discarded.